### DISCUSSION TITLE
combine prediction of slices using SAHI from external prediction result in txt file

### DISCUSSION THREAD

**Discussion by ramdhan1989** [obss/sahi#612] [Ideas]
Hi,
I would like to ask if I can use SAHI to combine prediction of slices that are stored in txt file? I did detection of objects (in smaller tiles) outside SAHI and return the result in the form txt file with the name of the file  for example 14_0_4_2052_4313_2564_3801 where :
14 = name of file
0=row position in original image 
4=columns position in original image
2052_4313_2564_3801 = XYXY position relative in original image
the content of txt file as follow : 
class bbox(yolo format) confidence

please advise
thank you

**Comment by fcakyon** [comment]
@ramdhan1989 thats a great usage! Unfortunately, SAHI currently cannot handle that.

SAHI automatically slices images with this function: https://github.com/obss/sahi/blob/b93132d175a729079f526452587fa7fb1df5f1c9/sahi/slicing.py#L243 and used that data during sliced prediction: https://github.com/obss/sahi/blob/b93132d175a729079f526452587fa7fb1df5f1c9/sahi/predict.py#L198

You can tweak it a bit so that it reads from txt instead of this function 👍 

We are open to any PR 💯
